modules/nf-core/genomad/download/genomad-download.diff

```diff
@@ -0,0 +1,14 @@
+Changes in module 'nf-core/genomad/download'
+--- modules/nf-core/genomad/download/main.nf
++++ modules/nf-core/genomad/download/main.nf
+@@ -4,7 +4,7 @@
+     conda "bioconda::genomad=1.5.2"
+     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+         'https://depot.galaxyproject.org/singularity/genomad:1.5.2--pyhdfd78af_0':
+-        'biocontainers/genomad:1.5.2--pyhdfd78af_0' }"
++        'quay.io/biocontainers/genomad:1.5.2--pyhdfd78af_0' }"
```

Comment:
So that isnt updated yet in the pipeline? I thought this is supposed to be not handled by `singularity.registry`  ?

Reply:
@CarsonJM @PhilPalmer can uo comment on this? Any reason why the patch reverting?

Reply:
For some reason when I was adding this module, the tests were not passing until I made the patch. Here was the comment you left prior to merging (https://github.com/nf-core/mag/pull/364#discussion_r1255350233)

Reply:
Aha! Well future James forgot pastbJames. Thanks for the reminder!

Reply:
Aha! Well future James forgot past James. Thanks for the reminder!

Reply:
@jfy133 Is there anything you would like me to do to help with this PR? I am happy to review it and help make changes if needed!

Reply:
Sorry for the delay, short break form parentla leave for a conference... so no difference in time availablility :sweat_smile: 

I think we are good for now @CarsonJM thank you! the changes in the comments are not too complicated.
 It will be monitoring issues/slack reports for bugs when the release is out (and then rapid subsequent releases after) that will be where help would benefit most <3

Reply:
I'm happy to help where I can! Sorry I have been less responsive lately, I have my parental leave coming up soon so I have been frantically trying to finish up all my wet-lab work before then! 😅